.pre-commit-hooks.yaml

```diff
@@ -1,7 +1,21 @@
 - id: ruff-conda
   name: ruff-conda
   description: "Run 'ruff' for extremely fast Python linting"
-  entry: ruff --fix --exit-non-zero-on-fix
+  entry: ruff check --force-exclude
```

Comment:
Not a fan of this. All our other pre-commit hooks are auto-fixing :eyes:

Reply:
There are quite a few rules for which auto-fixing is available (e.g. superfluous imports, unused variables, ...)

Reply:
Alright, then let's use `ruff --fix --exit-non-zero-on-fix`